Automattic/newspack-blocks
# Logo Update

### All Submissions:






### Changes proposed in this Pull Request:

Update Newspack logo in the block picker. 

<img width="429" alt="Screen Shot 2019-08-21 at 6 37 36 PM" src="https://user-images.githubusercontent.com/1477002/63473307-fba08f00-c442-11e9-89ac-abe3a41310e4.png">

### How to test the changes in this Pull Request:

1. Check out branch, run `npm run build:webpack`
2. Activate `Newspack Blocks` plugin, deactivate `Newspack Ads`, if necessary. 
3. Create a post, open the block picker, scroll to the bottom.
4. Observe revised Newspack logo.

### Other information:
